src/app/core/admin/admin-menu/admin-menu-item-details/admin-menu-item-details.component.ts

```diff
@@ -64,13 +71,24 @@ export class AdminMenuItemDetailsComponent implements OnInit {
   private loadAvailableRoutes(): { value: string; label: string }[] {
     const allConfigs: ViewConfig[] =
       this.configService.getAllConfigs<ViewConfig>(PREFIX_VIEW_CONFIG);
-    return allConfigs
-      .filter((view) => !view._id.includes("/:id")) // skip details views (with "/:id" placeholder)
-      .map((view) => {
-        const id = view._id.replace(PREFIX_VIEW_CONFIG, "/");
-        const label = view.config?.entityType?.trim() || view.component || id;
-        return { value: id, label };
+
+    let availableViews = allConfigs.filter(
+      (view) => !view._id.includes("/:id"),
+    ); // skip details views (with "/:id" placeholder)
+
+    // For shortcuts, exclude entityType routes
+    // For admin menu items, show all routes (no filtering)
+    if (this.isShortcut) {
+      availableViews = availableViews.filter((view) => {
+        return !view.config?.entityType;
       });
+    }
```

Comment:
We keep repeating ourselves around this:
No special filtering for the routes dropdown for shortcuts, please.

```suggestion
```